package.json

```diff
@@ -480,6 +481,7 @@
     "strip-ansi": "^3.0.1",
     "supertest": "^3.1.0",
     "supertest-as-promised": "^4.0.2",
+    "tape": "^4.13.0",
```

Comment:
is this dependency forgotten here? 😃

Reply:
It's used in the new file `test/harden/child_process.js`

Reply:
I miss that 👍